python/src/fastfec/utils.py

```diff
@@ -76,44 +87,58 @@ def find_fastfec_lib():
 
 
 def as_bytes(text):
-    """Converts text to bytes, or leaves intact if already bytes or none"""
+    """
+    Converts text to bytes, or leaves intact if already bytes or none
+    """
     if isinstance(text, str):
         # Convert to bytes if in string form
-        return text.encode('utf8')
+        return text.encode("utf8")
```

Comment:
We technically can omit the param since it's a default

Reply:
(I know this is code I wrote)

Reply:
Out of scope for this PR IMO, I just ran the formatter